src/components/mapPageComponents/mapView/index.tsx

```diff
@@ -202,48 +201,43 @@ const MapView: React.FC<MapViewProps> = ({ blocks, neighborhoods }) => {
           }
           return name;
         }
-        */
+
         // Loads the objects into the layer
         neighborhoodsLayer.addGeoJson(neighborhoods);
-        /*
-          () => {
-            // For each feature in neighbourhoodsLayer, add a marker
-            // We need to do it here as the GeoJson needs to load first
-            neighborhoodsLayer.forEach((feature) => {
-              // If you want, check here for some constraints.
-              const marker = new google.maps.Marker({
-                map,
-                draggable: false,
-                label: {
-                  color: 'white',
-                  fontWeight: 'bold',
-                  text: shortHand(feature.getProperty('Name')),
-                },
-                // Removed the icon here, only text on map.
-                icon: {
-                  labelOrigin: new google.maps.Point(11, 50),
-                  url: '',
-                  size: new google.maps.Size(22, 40),
-                },
-                position: {
-                  lat: feature.getProperty('Latitude'),
-                  lng: feature.getProperty('Longitude'),
-                },
-              });
-              markersArray.push(marker);
-              marker.setMap(map);
-            });
-          },
-        );
-        */
+        neighborhoodsLayer.forEach((feature) => {
+          // For each feature in neighbourhoodsLayer, add a marker
+          // We need to do it here as the GeoJson needs to load first
+          // If you want, check here for some constraints.
+          const marker = new google.maps.Marker({
+            map,
+            draggable: false,
+            label: {
+              color: 'white',
```

Comment:
⛏️ Is there a way to do a shadow or an outline? The text can be hard to read on the lighter areas of the map